lib/data/service/lesson_service.dart

```diff
@@ -26,4 +27,32 @@ class LessonServices {
 
     return bookList.map((book) => BookModel.fromJson(book as Map<String, dynamic>)).toList();
   }
+
+  Future<List<ChapterModel>> fetchChapters(int bookId) async {
+    final HttpLink httpLink = HttpLink('https://api-preprod.lelivrescolaire.fr/graph');
+
+    final GraphQLClient client = GraphQLClient(
+      cache: GraphQLCache(),
+      link: httpLink,
+    );
+
+    final QueryOptions options = QueryOptions(
```

Comment:
In JS, I can provide a generic type argument to the `query` function that povides type to my QueryResult (and the variables). Can you do the same in Dart?